### PR TITLE
Schedule x-option registration in didInsertElement() hook instead of didRender()

### PR DESCRIPTION
**Pull request by arunasf**
Just a simple hook rename didRender() -> didInsertElement()
See #70

### CODE REVIEW

File: addon/components/x-option.js [hunk 1]
```diff
@@ -51,7 +51,8 @@ export default Ember.Component.extend({
    *
    * @override
    */
-  didRender() {
+  didInsertElement() {
+    this._super();
```
Comment (Robdel12) [inline addon/components/x-option.js]: Can you also pass in `this` and `arguments`?

```js
this._super(this, arguments);
```

**Comment by Robdel12** [comment]
Thanks for the PR (and issue!) After that one small change we should be good to go :+1:

**Comment by arunasf** [comment]
Sorry for double commit :) thought ```--amend``` should do the trick, but somehow it didn't work

**Comment by Robdel12** [comment]
It's all good!

**Comment by arunasf** [comment]
Just checking ... the line should be ```this._super.apply(this, arguments);``` , not ```this._super(this, arguments);```.

Another PR?

**Comment by Robdel12** [comment]
Oh yea, that's right. That was my bad.

**Comment by arunasf** [comment]
Shit happens then u let Java guy to mess with JS :) Sorry for problems, next time i'll try better.

Anyway. Should I make another PR or you'll just fix it yourself?

**Comment by Robdel12** [comment]
I can fix it. Haha it was my fault, I suggested the wrong code in my comment. Sorry about that!

**Comment by arunasf** [comment]
Thanks.

**Pull request merged**